src/app/space/create/apps/components/app-card.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="card-pf" (click)="collapsed = !collapsed">
+  <div class="card-pf-body">
+    <span class="pficon pficon-ok"></span>
+    <span id="podCountLabel">{{ podCount | async }} Pods</span>
+    <span id="versionLabel">{{ version | async }}</span>
+    <div class="pull-right dropdown-kebab-pf dropdown" dropdown (click)="$event.stopPropagation()">
+      <button class="btn btn-link pull-right dropdown-toggle" id="kebab" type="button" aria-haspopup="true"
+        aria-expanded="true" data-toggle="dropdown" dropdownToggle>
+        <span class="fa fa-ellipsis-v"></span>
+      </button>
+      <ul class="dropdown-menu-right dropdown-menu" aria-labelledby="kebab" role="menu" *dropdownMenu>
+        <li>
+          <a>Action</a>
```

Comment:
does the `<a>` have an action associated with it, or just styling?

Reply:
It's just a placeholder for now, hence the lack of associated action and generic label of "Action".